roles/network_plugin/ovn4nfv/templates/ovn4nfv-k8s-plugin.yml.j2

```diff
@@ -556,7 +556,7 @@ spec:
       hostNetwork: true
       hostPID: true
       nodeSelector:
-        beta.kubernetes.io/arch: amd64
+        {{ 'beta.' if kube_version is version('v1.18.0', '<') }}kubernetes.io/os: linux/arch: amd64
```

Comment:
Why isn't here `kubernetes.io/arch: amd64` instead of `kubernetes.io/os: linux/arch: amd64` ?
`

Reply:
my mistake, copy/paste is evil )
thank you for your attention